scripts/docker/buildpack-deps/Dockerfile.ubuntu2004

```diff
@@ -48,7 +48,7 @@ FROM base AS libraries
 # EVMONE
 RUN set -ex; \
 	cd /usr/src; \
-	git clone --branch="v0.8.0" --recurse-submodules https://github.com/ethereum/evmone.git; \
+	git clone --branch="v0.9.1" --recurse-submodules https://github.com/ethereum/evmone.git; \
```

Comment:
What about the other two dockerfiles? They still have 0.8.2 hard-coded.

Reply:
Which other two?

I think ossfuzz was missed, but that's not even using it? Will fix that.